static/locales/ja.json

```
@@ -1463,6 +1479,7 @@
         "unregisterVoiceTitle": "ボイス登録を解除",
         "voiceUnregistered": "ボイス登録が解除されました",
         "unregisterFailed": "登録解除に失敗しました",
+        "partialSaveVoiceFailed": "キャラクターは保存されましたが、音声の更新に失敗しました: {{error}}",
```

Biome (2.4.10)

[error] The key partialSaveVoiceFailed was already declared.

(lint/suspicious/noDuplicateObjectKeys)